pkg/minikube/registry/drvs/qemu2/qemu2.go

```diff
@@ -110,8 +110,15 @@ func qemuVersion() (semver.Version, error) {
 	if err != nil {
 		return semver.Version{}, err
 	}
-	v := strings.Split(strings.TrimPrefix(string(rr), "QEMU emulator version "), "\n")[0]
-	return semver.Make(v)
+	versionPattern :=`(\d+\.\d+\.\d+)`
```

Comment:
how about adding this to a func and then add unit test for this in qemu2_test.go ?

Reply:
that is obviously a good proposal, I am quite busy atm, this will delay the PR

Reply:
it would make sense to get representative version strings from the brew version of qemu (I do not use brew) do you have that available?

Reply:
no rush, let me know if you still interested, I think without unit test this could be cause issues, we aiming to release sep 1st